picard/formats/id3.py

```diff
@@ -365,7 +365,7 @@ def _save(self, filename, metadata):
                     if frameid == 'WCOP':
                         # Only add WCOP if there is only one license URL, otherwise use TXXX:LICENSE
                         if len(values) > 1 or not valid_urls:
-                            tags.add(id3.TXXX(encoding=encoding, desc=self.__rtranslate_freetext[name], text=values))
+                            tags.add(id3.TXXX(encoding=encoding, desc=self.__rtranslate_freetext[name], text=[' / '.join(values)]))
```

Comment:
Instead of repeating this 5 times, i would create a local function after line 283:

```python
def addTXXX(desc, values):
  tags.add(id3.TXXX(encoding=encoding, desc=desc, text=[' / '.join(values)]))

...
addTXXX(desc=self.__rtranslate_freetext[name], text=values)
```

The separator should be configurable per option imho.